core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -292,7 +293,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                     FRIENDS,
                     NOTIFICATIONS,
                     HABIT_ASSIGN_ID + "/friends/habit-duration-info",
-                    "/ai/**")
+                    "/ai/**",
+                    SETTINGS_EXPORT)
```

Comment:
Why do we need this endpoint for users? 
Shouldn't it be available only to admins and moderators?